### PR TITLE
`alpha init`: prompt chain

### PR DESCRIPTION
**Pull request by YaroShkvorets**
Implements #255

- If Ethereum protocol selected - prompt chain
- Mainnet/BNB/Polygon/Goerli/Sepolia available

### CODE REVIEW

File: codegen/templates/ethereum/Makefile [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-STOP_BLOCK ?= +100
 
 .PHONY: build
 build:
 	cargo build --target wasm32-unknown-unknown --release
 
 .PHONY: run
 run: build
-	substreams run -e $(ENDPOINT) substreams.yaml map_events -s $(START_BLOCK) -t $(STOP_BLOCK)
+	substreams run -e $(ENDPOINT) substreams.yaml map_events $(if $(START_BLOCK),-s $(START_BLOCK)) $(if $(STOP_BLOCK),-t $(STOP_BLOCK))
```
Comment (maoueh) [inline codegen/templates/ethereum/Makefile:9]: Nice

File: codegen/templates/ethereum_chain.go [hunk 2]
```diff
@@ -1,15 +1,55 @@
...
+		DefaultContractName:    "GETH Token",
+		FirehoseEndpoint:       "goerli.eth.streamingfast.io:443",
+	},
+	"Sepolia": {
+		DisplayName:            "Sepolia Testnet",
+		ExplorerLink:           "https://sepolia.etherscan.io",
+		ApiEndpoint:            "https://api-sepolia.etherscan.io",
+		DefaultContractAddress: "0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984",
+		DefaultContractName:    "UNI Token",
+		FirehoseEndpoint:       "sepolia.streamingfast.io:443",
```
Comment (maoueh) [inline codegen/templates/ethereum_chain.go]: Can you temporary comment this one, we don't have an endpoint available for this yet.

**Review by maoueh — APPROVED** [review]
Approving because everything else is perfect!

File: codegen/templates/ethereum_chain.go (hunk 2 above)
Comment (YaroShkvorets) [reply codegen/templates/ethereum_chain.go]: I can use Pinax endpoint for Sepolia if that's OK with you: sepolia.firehose.pinax.network:9000

Comment (maoueh) [reply codegen/templates/ethereum_chain.go]: I thought about this but the auth is different and this is not documented today, so for now, let's just keep it commented out.

Comment (YaroShkvorets) [reply codegen/templates/ethereum_chain.go]: Removed Sepolia

**Pull request merged**